contrib/checkbox-provider-ce-oem/series_classic22/snap/snapcraft.yaml

```
@@ -0,0 +1,100 @@
+name: checkbox-ce-oem
+summary: Checkbox CE OEM test runner and public providers for 22.04 classic
+description: "Checkbox CE OEM test runner and public providers for 22.04 classic"
+confinement: classic
+grade: stable
+
+version: '0.12-jammy'
```

How about starting the version from `1.0`?
```suggestion
version: '1.0-jammy'
```